content/en/blog/8-must-know-serverless-terms.md

```diff
@@ -0,0 +1,158 @@
++++
+title = "8 Must Know Serverless Terms For A Developer"
+date = "2022-03-01T11:30:34+05:30"
+author = "Atulpriya Sharma"
+description = "Planning to start your cloud native journey? Here are 8 Must Know Serverless Terms For A Developer."
```

Comment:
Are you building a serverless application? Here are 8 must know terms for a serverless developer

Reply:
Done.